projects/canopy/src/lib/forms/forms.module.ts

```diff
@@ -17,6 +18,7 @@ import { LgValidationModule } from './validation/validation.module';
     LgInputModule,
     LgLabelModule,
     LgRadioModule,
+    LgFilterButtonModule,
```

Comment:
For anyone else reviewing we had an interesting discussion around wether to deprecate this LgFormsModule. It's useful in that it's just one module to import, but as it grows in size people may end up increasing their bundle size with form components they aren't using. Unless of course the tree shaking is more clever than I think.

Reply:
I definitely agree on the `LgFormsModule`. I can't see any need for it.